src/beamlime/kafka/helpers.py

```diff
@@ -32,3 +32,15 @@ def detector_topic(instrument: str) -> str:
     Return the topic name for the detector data of an instrument.
     """
     return topic_for_instrument(topic='detector', instrument=instrument)
+
+
+def source_name_separator() -> str:
+    """
+    Return the separator between device name and signal name.
+
+    This is used to construct the source name from the device name and signal name
+    The source_name is used in various Kafka messages.
+
+    ':' is used as the separator in the ECDC naming convention at ESS.
+    """
+    return ':'
```

Comment:
Do we need to have a separate function here?

Reply:
You mean, we should just hard-code it where used?

Reply:
Yeah, seems like an overkill and too much indirection unless this needs to be controlled or changed in the future a lot.

Reply:
But it avoids 5 hours debugging because a wrong separator made it into some random file?

Reply:
But maybe it would be better to have a helper function building the source name, instead of one for the separator?

Reply:
> But maybe it would be better to have a helper function building the source name, instead of one for the separator?

Yeah I do like that better, but anyway we can iterate on that later.